### DISCUSSION TITLE
how to change default selected component style

### DISCUSSION THREAD

**Discussion by phanix1998** [GrapesJS/grapesjs#4430] [Q&A] [answered]
Hello,
I need to change default style of selecte component,

.gjs-selected {
    outline: 2px solid #3b97e3 !important;
    outline-offset: -2px;
}

this is the default style, how can I change this? Thanks!

**Comment by artf** [comment, marked as answer]
Check this issue: https://github.com/artf/grapesjs/issues/3515#issuecomment-864752465

**Reply by phanix1998** [reply]
Thanks!
